### PR TITLE
Bug fix: when canceling flows, we should send out Killed events corre…

### PR DESCRIPTION
**Pull request by byjiang1996**
…ctly

### CODE REVIEW

File: azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java [hunk 1]
```diff
@@ -560,6 +557,8 @@ protected void handleCancelFlow(ExecutionReference executionReference,
       ExecutionControllerUtils.finalizeFlow(this.executorLoader, this.alerterHolder,
           executableFlow, "Cancel action has been called but the flow is unreachable.", null,
           finalizingStatus);
+      this.fireEventListeners(Event.create(executableFlow,
```
Comment (hqin2016) [inline azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java:560]: can you add comment to the code changes?

File: azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java [hunk 2]
```diff
@@ -574,6 +573,8 @@ protected void handleCancelFlow(ExecutionReference executionReference,
       final String finalizingReason = "Unable to gracefully kill the flow execution.";
       ExecutionControllerUtils.finalizeFlow(this.executorLoader, this.alerterHolder,
           executableFlow, finalizingReason, e, finalizingStatus);
+      this.fireEventListeners(Event.create(executableFlow,
```
Comment (hqin2016) [inline azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java:576]: same as above, comment is needed to describe the code change.

File: azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java (hunk 1 above)
Comment (byjiang1996) [reply azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java:562]: Done. Thx!

File: azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java (hunk 2 above)
Comment (byjiang1996) [reply azkaban-common/src/main/java/azkaban/executor/AbstractExecutorManagerAdapter.java:580]: Done. Thx!

**Review by hqin2016 — APPROVED** [review]
Thanks for the bug fix.

**Pull request merged**